kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/PositionService.java

```diff
@@ -114,7 +114,7 @@ public interface PositionService {
     public String getLastSentence();
 
     /**
-     * Returns the GNSS System used to get the position information. Could be one or more system (eg: GPS + GLONASS).
+     * * Returns the GNSS System used to get the position information @reference GNSSType. Could be one or more system (eg: GPS + GLONASS).
```

Comment:
Extra `*` character?

```suggestion
     * Returns the GNSS System used to get the position information @reference GNSSType. Could be one or more system (eg: GPS + GLONASS).
```